### ISSUE TITLE
characters idle after player death (from FPSC ROMAN pack) 

### ISSUE THREAD

**Issue by Daveutopias** [TheGameCreators/GameGuruRepo#286]
Hi, with the new game guru version i have noticed if i put more characters with the same animation and i start a test level they work fine until the player Death, they stuck his animations, while before new version this do NOT happen.

they have script classics\ai_classiccharacter_roman.lua (maybe they need to be fix like was done with drunken standard character?)

I am not sure if this not happen with stock characters.

**Comment by LeeBamberTGC** [comment]
Can you point to the exact location of the entity and the entity name, and a small step by step of what you did and what you saw, and what you expected to see, thanks.

**Comment by Teabone3** [comment]
I'm assuming you are referencing the roman characters from FPSC converted for GG, acquired from my site here? http://freetoronto.org/gameguru

The scripts are very outdated. I will see what I can do if i ever get the time to. But this isn't so much a github matter as these characters are not stock with GG. There are some legacy issues that I need to resolve to get these old characters to behavior properly. I have just been incredibly busy and lack a bit of the old motivation i had back in the day. I may get around to it some day.

**Comment by bonesy23438** [comment]
yeah same here it does it with custom characters and stock characters
![Desktop Screenshot 2019 06 26 - 08 34 21 49](https://user-images.githubusercontent.com/37757086/60171341-5b433b00-9802-11e9-9952-1103ef31968d.png)

**Comment by LeeBamberTGC** [comment]
@bonesy23438 Any chance you can provide an FPM that uses a stock character and tell me what I do in the level to see this issue, cheers!

**Comment by bonesy23438** [comment]
Hi Lee
Sorry for the delay but have zipped up an fpm with stock characters and
assets to show bug.While i only managed to get it to happen twice i couldnt
take a screenshot as my geforce experience wasnt working properly,but it
seems to be when an explosion happens nearby in its radius.So ive put some
custom barrels and a modified wepon with projectiles with explosion to see
if it happens.Like i said ive tested a few times and it only happened a few
times.I have looked on Github to see if issue is closed and i cant see that
it has.
Thanks Andy
 Idle_animation_test.zip
<https://drive.google.com/file/d/1ywVNnXxg6ijg2TPBJJJ6zTKk2KmusIkc/view?usp=drive_web>

**Comment by LeeBamberTGC** [comment]
@bonesy23438 I checked the zip file and it contains a custom weapon but this does not seem to relate to the report of "if i put more characters with the same animation and i start a test level they work fine until the player Death". I played the map (with a stock weapon) and it played fine, no animation idles. The characters all used "ai_soldier.lua" it seems, so no issues with this one, Probably fixed.
